main.go

```diff
@@ -76,12 +76,6 @@ func overrideFromFlags(c *cli.Context, config *server.Config) error {
 	if c.IsSet("tls-key") {
 		config.TLSKey = c.String("tls-key")
 	}
-	if c.IsSet("tls-client-auth") {
-		config.TLSClientAuth = c.Bool("tls-client-auth")
-	}
-	if c.IsSet("tls-client-auth-ca") {
-		config.TLSClientAuthCA = c.String("tls-client-auth-ca")
-	}
```

Comment:
My initial logic was:

- Since Auth and Authz involves different configuration that go together, it is better to "force" users to use config file, since it is more verbose and easier to understand.

But indeed, that would be too harsh and it is a breaking change, so I re-introduce them back.

However, authz configuration does not provide command-line config. 

To be coherent, I will add also config flag for Authorization as well

Reply:
In fact, I have just realized that we don't actually even support these flags yet. Let's take a look https://github.com/liftbridge-io/liftbridge/blob/master/main.go#L141

It seems that there is no way even at the moment, to set `tls-client-auth-ca` or `tls-client-auth` via the cli.

I can add support for them, but IHMO, it is better just to "force" user use the config file, since authz and auth configs are rather complex . What do you think ?